app-ng/src/package.lisp

```diff
@@ -3,4 +3,23 @@
 ;;;; Author: Robert Smith
 
 (defpackage #:qvm-app-ng
-  (:use #:cl #:qvm))
+  (:use #:cl #:qvm)
+
+  ;; job.lisp
+  (:export
+   #:job-status
+   #:job-status-fresh
+   #:job-status-running
+   #:job-status-interrupted
+   #:job-status-finished
+   #:job-status-error
+   #:job-status-name
+   #:make-job
+   #:job-running-p
+   #:job-interrupted-p
+   #:job-finished-p
+   #:job-start
+   #:kill-job
+   #:force-kill-job
```

Comment:
Don't forget to remove or rename here as well. Probably remove, since exporting unsafe functions feels wrong.